admin/data/sql/migrations/024.do.create-vewCheckWindowsWithFormCount.sql

```diff
@@ -0,0 +1,8 @@
+CREATE VIEW [mtc_admin].[vewCheckWindowsWithFormCount] AS
```

Comment:
s/vew/view/ ?

Reply:
that's pure habit from using `tbl`, `vew` and `spr` for the last 20 years 😆

Reply:
Okay, np